oom-frontend/src/app/shared/modules/maintainer/maintainer-editor/maintainer-editor.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { MaintainerEditorComponent } from './maintainer-editor.component';
+
+describe('MaintainerEditorComponent', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Out-Of-Memory-Team/out-of-memory/pullRequest?prid=7533382)
```suggestion
describe("MaintainerEditorComponent", () => {
```